src/main/kotlin/apply/infra/mail/SimpleMailSender.kt

```diff
@@ -19,4 +22,23 @@ class SimpleMailSender(
         }
         mailSender.send(message)
     }
+
+    override fun sendBCC(
+        toAddresses: Array<String>,
+        subject: String,
+        body: String,
+        files: Map<String, ByteArrayResource>
+    ) {
+        val message = mailSender.createMimeMessage()
+        val mimeMessageHelper = MimeMessageHelper(message, true).apply {
+            setFrom(mailProperties.username)
```

Comment:
확인을 위해 받은 사람에 우테코 메일을 지정하면 어떨까요? 50통당 한 번씩 받는 건 무섭긴 하네요.

```suggestion
            setFrom(mailProperties.username)
            setTo(mailProperties.username)
```

Reply:
이 부분은 마지막에 결정하고 다시 댓글 달도록 하겠습니다.